doc/dev/release-procedure.md

```diff
@@ -62,10 +62,13 @@ cp sonata-vX.Y.bit.slot3.uf2 ../
 
 ### Automated testing
 
-Run tests on FPGA (note python3.11 or above required), adjust the /dev part to the UART as required.
+Run tests on simulation (note python3.11 or above required). FPGA testing is currently done in CI and requires quite a few add-ons to your board.
```

Comment:
For a release we should definitely repeat the tests on FPGA.

We do need to document the physical add-ons needed to successfully run the tests, though maybe not here as it's got wider applicability.